src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1087,68 +1087,45 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
 
             if (Form is not null)
             {
-                // Content headers will be set by MultipartFormDataContent which will throw unless we clear them first
-                WebSession.ContentHeaders.Clear();
-
```

Comment:
This is not used in the context but can have a side effect. Please revert.

Reply:
I did not remove it, I moved it inside the `SetRequestContent(HttpRequestMessage request, MultipartFormDataContent multipartContent)`, it should work the same way